targets/xu5_st1/target-top.dts

```diff
@@ -3,10 +3,19 @@
 
 &sdhci1 {
 	disable-wp;
+	no-1-8-v;
```

Comment:
the 'no-1-8-v' is a strange one, as I had previously noticed that it didn't seem to need to be included when using Vivido-2023.2, despite being needed before: https://github.com/PandABlocks/PandABlocks-FPGA/pull/174

Reply:
That's strange, I built master in both repos and I needed this fix